### ISSUE TITLE
Add default plot for classif.glmnet/classif.cv_glmnet/regr.glmnet/regr.cv_glmnet/surv.glmnet/surv.cv_glmnet

### ISSUE THREAD

**Issue by mllg** [mlr-org/mlr3viz#35]
There is a base R plot generic:
![glmnet](https://user-images.githubusercontent.com/1260920/89382703-8d101c00-d6fb-11ea-95c6-f4d31b22ad5c.png)

**Comment by missuse** [comment]
ggfortify has autoplot methods for glmnet and cv.glmnet objects:

```
task = tsk("mtcars")
learner = lrn("regr.cv_glmnet")
learner$train(task)

autoplot(learner$model)
```
![cv_glmnet](https://user-images.githubusercontent.com/35730264/89646588-559c9d80-d8bc-11ea-8fa4-4a4e89d88587.png)

```
task = tsk("mtcars")
learner = lrn("regr.glmnet")
learner$train(task)

autoplot(learner$model)
```
![glmnet](https://user-images.githubusercontent.com/35730264/89646695-8aa8f000-d8bc-11ea-9152-81570504663d.png)

**Comment by mllg** [comment]
Implemented in 1346f326ee18af341d98734705d803a6cc9f234c.
